trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-summary-helper.js

```diff
@@ -23,6 +25,7 @@ const getReportReconciliationSummaryViewModel = (summaryApiResponse) =>
   summaryApiResponse.map((item) => ({
     ...item,
     displayStatus: reconciliationStatusCodeToDisplayStatus[item.status],
+    formattedDateUploaded: item.dateUploaded ? format(parseISO(item.dateUploaded), 'd MMM yyyy') : undefined,
```

Comment:
I think this can be shortened to `item.dateUploaded && format(parseIso(item.dateUploaded), 'd MMM yyyy');`

Reply:
I think in most cases this would result in what we want, but it doesn't behave exactly the same in every case. e.g. if `item.dateUploaded` was the empty string then
```
item.dateUploaded && format(parseISO(item.dateUploaded)
```
would result in `''` instead of `undefined`.